src/lib/browse/BusRoutesLayerControl.svelte

```diff
@@ -0,0 +1,77 @@
+<script lang="ts">
+  import type { MapGeoJSONFeature } from "maplibre-gl";
+  import {
+    hoveredToggle,
+    overwriteLineLayer,
+    overwritePmtilesSource,
+  } from "../../maplibre_helpers";
+  import { map } from "../../stores";
+  import {
+    ColorLegend,
+    ExternalLink,
+    HelpButton,
+    InteractiveLayer,
+  } from "../common";
+  import { Checkbox } from "../govuk";
+  import { colors } from "./colors";
+
+  let name = "bus_routes";
+
+  overwritePmtilesSource(
+    $map,
+    name,
+    `https://atip.uk/layers/v1/${name}.pmtiles`
+  );
+
+  overwriteLineLayer($map, {
+    id: name,
+    source: name,
+    sourceLayer: name,
+    color: [
+      "case",
+      ["boolean", ["get", "has_bus_lane"], false],
+      colors.bus_route_with_lane,
+      colors.bus_route_without_lane,
+    ],
+    width: 5,
+    opacity: hoveredToggle(0.5, 1.0),
+  });
+
+  let show = false;
+
+  function tooltip(feature: MapGeoJSONFeature): string {
+    if (feature.properties.has_bus_lane) {
+      return "<p>At least one bus route crosses here, with a bus lane in one or more directions</p>";
```

Comment:
Is the tooltip too verbose and annoying?

Reply:
I don't think so, in any case one to await feedback: it doesn't really matter whether we think it's verbose or annoying!